### PR TITLE
Drain training with opni-normal-intervals

### PR DESCRIPTION
**Pull request by sanjay920**
Needs to create opni-normal-intervals ES index first:
https://github.com/rancher/opni-kibana-dashboard/pull/9

### CODE REVIEW

File: drain-service/drain_training_inferencing.py [hunk 1]
```diff
@@ -294,44 +293,57 @@ def weighted_avg_and_std(values, weights):
...
+                training_start_ts_ms = int(
+                    pd.to_datetime("now", utc=True).timestamp() * 1000
+                )
 
             if weighted_vol > 0.155 and not train_on_next_chance and stable:
-                training_end_ts_ns = time.time_ns()
+                training_end_ts_ms = int(
+                    pd.to_datetime("now", utc=True).timestamp() * 1000
+                )
                 normal_periods.append(
```
Comment (AmartC) [inline drain-service/drain_training_inferencing.py:307]: Why do we still need the normal_periods list? Only the start and end timestamp are being sent to DRAIN now so no need to have this normal_periods list anymore.

File: drain-service/drain_training_inferencing.py [hunk 2]
```diff
@@ -294,44 +293,57 @@ def weighted_avg_and_std(values, weights):
...
+                        {"start_ts": training_start_ts_ms, "end_ts": training_end_ts_ms}
                     )
                 train_payload = {
-                    "model_to_train": "nulog",
-                    "time_intervals": normal_periods,
+                    "start_ts": training_start_ts_ms,
+                    "end_ts": training_end_ts_ms,
                 }
+                try:
+                    await es.index(index="opni-normal-intervals", body=train_payload)
```
Comment (AmartC) [inline drain-service/drain_training_inferencing.py:331]: Where are you creating the opni-normal-intervals index? Don't you need to first create create the index before you place payload into the index, like this?
```
es.indices.create(index="opni-normal-intervals", ignore=400)
```

**Review by AmartC — CHANGES_REQUESTED** [review]
Couple clarifications required.

File: drain-service/drain_training_inferencing.py (hunk 2 above)
Comment (sanjay920) [reply drain-service/drain_training_inferencing.py:331]: It's set up here: https://github.com/rancher/opni-kibana-dashboard/pull/9

File: drain-service/drain_training_inferencing.py (hunk 1 above)
Comment (sanjay920) [reply drain-service/drain_training_inferencing.py:307]: we don't explicitly need it anymore. it does give us a good idea in the logs how frequently it is creating normal periods so I suggest we keep it

**Review by AmartC — APPROVED** [review]

**Pull request merged**